### ISSUE TITLE
Links to download pretrained model are not working. 

### ISSUE THREAD

**Issue by SamihaSara** [hasanirtiza/Pedestron#98]
I have tried to download pretrained models to run demo of Pedestron. But links are not working, Could you please check?
Also, Could you please check this issue https://github.com/hasanirtiza/Pedestron/issues/96#issuecomment-835696376?
Thanks.

**Comment by hasanirtiza** [comment]
True, some of the links are back (cascade(HR) on cityperson, cascade(MobNet) on cityperson, MGAN(VGG) on cityperson and cascade(HR) on ECP and Cascade(HR) on Crowdhuman). Remaining links will be back as well in 2 or 3 days hopefully. If there is a specific pre-trained model you want then let me know , I can prioritize that one.

**Comment by SamihaSara** [comment]
Thanks for helping. I am interested in Pre-trained models of Cascade Mask-R-CNN and  Faster R-CNN on CityPersons, EuroCity Persons, WiderPedestrian Challenge Dataset.

**Comment by WYJSJTU** [comment]
Thanks for helping. I also found the CSP on CityPersons is not available.

**Comment by hasanirtiza** [comment]
> Thanks for helping. I also found the CSP on CityPersons is not available.

CSP pre-trained model link is also back.

**Comment by ashep29** [comment]
Hi @hasanirtiza, it appears that the link to the RetinaNet model is broken. How do I access it?

**Comment by hasanirtiza** [comment]
Hi, we have updated the benchmarking table and you can see which models are available. For the moment, we do not have the link for pre-trained model of RetinaNet and I cannot provide a timeline. Perhaps, you can try the configs yourself and train a model that might be the fastest option as of now.

**Comment by ashep29** [comment]
Thanks @hasanirtiza. Could you please tell me the best way to change the nms from soft to greedy and/or change the iou threshold?

**Comment by hasanirtiza** [comment]
> Thanks @hasanirtiza. Could you please tell me the best way to change the nms from soft to greedy and/or change the iou threshold?

I suppose in the [config file](https://github.com/hasanirtiza/Pedestron/blob/d8a4065ffaaf493aee249a30ddf100425de13d90/configs/elephant/cityperson/ga_retinanet_ResNeXt101.py#L80). Did you try that ?
